src/datepicker/test/datepicker.spec.js

```diff
@@ -1832,6 +1834,17 @@ describe('datepicker directive', function () {
       expect(element.find('tbody').find('tr').length).toBeGreaterThan(3);
     });
 
+    it('has next and previous icons', function () {
+
+      expect($rootScope.nextIconClass).toEqual('glyphicon glyphicon-chevron-right');
+      expect($rootScope.prevIconClass).toEqual('glyphicon glyphicon-chevron-left');
+
+      var nextIconClassSelector = $rootScope.nextIconClass.split(' ').map(function (c) { return '.' + c; }).join('');
```

Comment:
I think that depends upon how you feel about `map` :stuck_out_tongue:. That being said, I think the best for readability would to just use `".glyphicon.glyphicon-chevron-right"` despite the repeatition.